plugin/src/sbt-test/sbtix/simple/test

```diff
@@ -1,5 +1,10 @@
 > compile
 > genNix
-$ exists repo.nix
+> reload plugins
+> genNix
+$ exec mv repo.nix repo-build.nix
```

Comment:
Why not just import repo.nix and project/repo.nix from default.nix instead? I'd like for the tests to also serve as documentation for end-users, and that carries less busywork.

Perhaps it'd make more sense to change genNix to emit a file named repo-build.nix, but hopefully repo-manual.nix won't be required for too long anyway. Also, what do you think about sbtix-repo.nix or sbtix-repo-auto.nix instead?